contracts/compliance/modular/modules/AbstractModule.sol

```diff
@@ -63,6 +63,9 @@
 pragma solidity 0.8.17;
 
 import "./IModule.sol";
+import "../IModularCompliance.sol";
```

Comment:
add the necessary imports at module level to avoid modifying audited contracts. We should modify the audited contracts only when it is absolutely necessary